### PR TITLE
Backup branch alexworkbench

### LINKED ISSUE #10: Create logo

### PR DESCRIPTION
**Pull request by AlexTerhar**
the final logo for the app
Closes #10

### CODE REVIEW

**Comment by PassiDel** [comment]
@AlexTerhar i applied the theme and logo to the login screen, there was a `TODO`

File: app/src/main/res/values/fridge_show_image_background.xml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <color name="fridge_show_image_background">#528540</color>
```
Comment (PassiDel) [inline app/src/main/res/values/fridge_show_image_background.xml:3]: Use color variable, or remove this whole file
```suggestion
    <color name="fridge_show_image_background">@color/fern_green</color>
```

**Review by PassiDel — APPROVED** [review]

**Comment by PassiDel** [comment]
I don't think we all need to approve this, IMHO it can be merged!

**Review by frcroth — APPROVED** [review]

**Review by henrijaeger — APPROVED** [review]
I agree with @PassiDel regarding his comment on the hard coded color 'fridge_show_image_background'.

Other than that everything looks fine to me!

**Pull request merged**